packages/itwin/measure-tools/src/api/Measurement.ts

```diff
@@ -285,6 +329,67 @@ export abstract class Measurement {
     this.onTransientIdChanged(prevId);
   }
 
+  public get drawingMetaData(): DrawingMetaData | undefined {
```

Comment:
Please add missing implementation in `Measurement.copyFrom`, `Measurement.readFromJSON`, `Measurement.writeToJSON`